code/modules/mob/living/damage_procs.dm

```diff
@@ -28,12 +28,12 @@
 			adjustStaminaLoss(damage * hit_percent)
 	return 1
 
-/mob/living/proc/apply_damage_type(damage = 0, damagetype = BRUTE) //like apply damage except it always uses the damage procs
+/mob/living/proc/apply_damage_type(damage = 0, damagetype = BRUTE, _required_status) //like apply damage except it always uses the damage procs
```

Comment:
Whats with _required_status it does absolutely nothing, and also breaks a few other things

Reply:
was being used to transfer required status to adjustbruteloss and adjustfireloss but it didnt work so I'll just use the boring way